src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -242,6 +244,7 @@ class ShenandoahHeap : public CollectedHeap {
   void increase_committed(size_t bytes);
   void decrease_committed(size_t bytes);
 
+  size_t bytes_allocated_since_gc_start();
```

Comment:
Do we really need this?

Reply:
Yes - this is used by the compact heuristic. Keeping this in minimizes changes to other heuristics.